flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamEdge.java

```diff
@@ -36,8 +36,8 @@
 
 	private final String edgeId;
 
-	private final StreamNode sourceVertex;
-	private final StreamNode targetVertex;
+	private final int sourceId;
+	private final int targetId;
```

Comment:
We usually name node id to vertexID， So I suggest that: 
```
	private final int sourceVertexId;
	private final int targetVertexId;
```
What do you think?

Reply:
These names are used mainly to consider consistency with the existing public methods getSourceId () and getTargetId ().

Reply:
Make sense.